src/browser/squirrel-update.coffee

```diff
@@ -43,11 +45,26 @@ spawn = (command, args, callback) ->
     error?.code ?= code
     error?.stdout ?= stdout
     callback?(error, stdout)
+  # This is necessary if using Powershell 2 on Windows 7 to get the events to raise
+  # http://stackoverflow.com/questions/9155289/calling-powershell-from-nodejs
+  spawnedProcess.stdin.end()
+
 
 # Spawn reg.exe and callback when it completes
 spawnReg = (args, callback) ->
   spawn(regPath, args, callback)
 
+# Spawn powershell.exe and callback when it completes
+spawnPowershell = (args, callback) ->
+  # set encoding and execute the command, capture the output, and return it via .NET's console in order to have consistent UTF-8 encoding
+  # http://stackoverflow.com/questions/22349139/utf-8-output-from-powershell
+  args[0] = "[Console]::OutputEncoding=[System.Text.Encoding]::UTF8\r\n$output=#{args[0]}\r\n[Console]::WriteLine($output)"
```

Comment:
Do you think it might look better to split this over multiple lines for readability like:

```coffee
args[0]  = [
  '[Console]::OutputEncoding=[System.Text.Encoding]::UTF8'
  "$output=#{args[0]}"
  '[Console]::WriteLine($output)'
].join('\r\n')
```